myia/info.py

```diff
@@ -206,3 +206,14 @@ def __exit__(self, type, value, tb):
         """Exit the context of this `About`."""
         top = _about.pop()
         assert isinstance(top, DebugInfo) and top.about is self
+
+
+__consolidate__ = True
+__all__ = [
+    'About',
+    'DebugInfo',
+    'DebugInherit',
+    'NamedDebugInfo',
+    'StackVar',
+    'current_info',
```

Comment:
Maybe current_info and StackVar are somewhat private.

Reply:
Could be useful to export `current_info`